src/solver/NSTempTurbConSolver.cpp

```diff
@@ -119,7 +116,7 @@ void NSTempTurbConSolver::do_step(real t, bool sync) {
     Field &kappa_t = m_field_controller->get_field_kappa();  // kappa_t - Eddy thermal diffusivity
     Field &gamma_t = m_field_controller->get_field_gamma();  // gamma_t - Eddy mass diffsusivity
 
-    size_t bsize = Domain::getInstance()->get_size(u.get_level());
+    size_t bsize __attribute__((unused))= Domain::getInstance()->get_size(u.get_level());
```

Comment:
used if turbulence is enabled. can be moved to the inquiry branch, where it is used. Furthermore, it can be simplified with `kappa_t` or `nu_t` `.get_size()`

Reply:
done in 9632d8b.
`bsize` was redefined in 193 and is used in 195
`bsize` was redefined in 258 and is used in 260.
no other occurrences.